arch/riscv/Kconfig.isa

```diff
@@ -111,6 +113,20 @@ config RISCV_ISA_EXT_ZIFENCEI
 	  provides explicit synchronization between writes to instruction
 	  memory and instruction fetches on the same hart.
 
+config RISCV_ISA_EXT_ZAAMO
+	bool
+	help
+	  (Zaamo) - Atomic memory operation subset of the A extension
+
```

Comment:
```suggestion
```

Reply:
This blank line is a convention in this file; see e.g. the entry for the M extension: https://github.com/zephyrproject-rtos/zephyr/blob/main/arch/riscv/Kconfig.isa#L25-L32